packages/arb-token-bridge-ui/src/components/TransferPanel/USDCDeposit/USDCDepositConfirmationDialog.tsx

```diff
@@ -74,81 +97,68 @@ export function USDCDepositConfirmationDialog(props: Props) {
   }))
 
   return (
-    <Dialog {...props}>
-      <div className="flex max-h-screen w-full flex-col md:w-[750px] lg:w-[925px]">
+    <Dialog
+      {...props}
+      title={`Move funds to ${destinationNetworkName}`}
+      actionButtonProps={{
+        disabled: !allCheckboxesCheched,
+        hidden: selectedTabName === SelectedTabName.ThirdParty
+      }}
+      onClose={handleActionButtonClick}
+    >
+      <div className="flex flex-col pt-4">
         <Tab.Group
           onChange={() => {
             setAllCheckboxesChecked(false)
           }}
         >
-          <div className="flex flex-row items-center justify-between bg-ocl-blue px-8 py-4">
-            <HeadlessUIDialog.Title className="text-2xl font-medium text-white">
-              Move funds to {destinationNetworkName}
-            </HeadlessUIDialog.Title>
-            <button
-              className="arb-hover"
-              onClick={() => {
-                props.onClose(false)
-              }}
+          <Tab.List className="flex border-b border-gray-dark">
+            <TabButton
+              aria-label="Arbitrum's bridge (USDC.e)"
+              onClick={() => setSelectedTabName(SelectedTabName.Bridged)}
             >
-              <XMarkIcon className="h-6 w-6 text-white" />
-            </button>
-          </div>
-          <Tab.List className="flex bg-ocl-blue">
-            <TabButton aria-label="Arbitrum's bridge (USDC.e)">
               Arbitrum&apos;s bridge (USDC.e)
             </TabButton>
-            <TabButton aria-label="Third party (USDC)">
+            <TabButton
+              aria-label="Third party (USDC)"
+              onClick={() => setSelectedTabName(SelectedTabName.ThirdParty)}
+            >
               Third party (USDC)
             </TabButton>
-            <TabButton aria-label="Circle (USDC)">Circle (USDC)</TabButton>
+            <TabButton
+              aria-label="Circle (USDC)"
+              onClick={() => setSelectedTabName(SelectedTabName.Cctp)}
+            >
+              Circle (USDC)
+            </TabButton>
           </Tab.List>
 
-          <Tab.Panel className="flex flex-col space-y-3 px-8 py-4">
-            <div className="flex flex-col space-y-6">
+          <Tab.Panel className="flex flex-col space-y-4 py-4">
+            <div className="flex flex-col space-y-4">
               <p className="font-light">
                 Receive{' '}
                 <span className="font-medium">Bridged USDC (USDC.e)</span> on{' '}
```

Comment:
We should link it to the block explorer's token page rather than bolding the text, it is a better UX so we should follow Figma here. Same for other tabs.